dev-db/spatialite/spatialite-5.0.1-r2.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
```

Comment:
Since we have to revision bump this anyway, maybe we can bump the EAPI to 8 as well.